lib/takeaway.rb

```diff
@@ -0,0 +1,90 @@
+require_relative 'restaurant.rb'
+require_relative 'order.rb'
+require_relative 'twilio_messager'
+require 'dotenv'
+
+class Takeaway
+  
+  attr_reader :restaurant, :order_log, :messager
+  
+  def initialize(restaurant = Restaurant.new, order=Order.new, messager=Messager.new) 
+    @action ||= nil 
+    @restaurant = restaurant 
+    @order_log  = order 
+    @messager = messager
+  end
+  
+  def menu
+    boot_strap
+    read_list
+    choice
+  end
+  
+  def choice
+    @action ? @action : @action = gets.chomp
+    display_menu
+  end
+  
+  def order(item, capacity=1) 
+    return "Please choose an item in the menu" unless item_exist?(item)
+    add_to_basket(item, capacity)
+    "#{capacity} x #{item.capitalize} added to your order."
+  end 
+  
+  def check_order
+    order_log.check_order
+  end
+  
+  def clear_order
+    order_log.clear_order
+  end
+  
+  def check_out(total)
+    return "The total is not correct or no order has been made!" if total_error(total)
```

Comment:
Metrics/LineLength: Line is too long. [86/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)